apps/condo/domains/common/components/PostMessageProvider/validators.ts

```diff
@@ -0,0 +1,19 @@
+import type { RequestParamValidator, AllRequestMethods } from './types'
+import Ajv from 'ajv'
+
+const ajv = new Ajv()
+
+const CondoWebAppResizeWindowParamsSchema = {
+    type: 'object',
+    properties: {
+        height: { type: 'number' },
```

Comment:
What about width?

Reply:
Width for iframes is always 100% for now